configDiffer/diff.py

```
@@ -0,0 +1,55 @@
+    end: int
+
+
+@dataclass
+class ParagraphDiff:
+    begin: int | None
+    end: int | None
+    line_diffs: dict[int, LineDiff]
+
+
+def diff_file(file_path1: Path, file_path2: Path) -> ParagraphDiff:
+    file1 = open(file_path1, "r").readlines()
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)